server/sonar-web/src/main/js/apps/component-measures/sidebar/DomainFacet.js

```diff
@@ -63,6 +63,15 @@ export default class DomainFacet extends React.PureComponent {
     return measureSelected || overviewSelected;
   };
 
+  getValues = () => {
+    const { domain, selected } = this.props;
+    const measureSelected = domain.measures.find(measure => measure.metric.key === selected);
+    const overviewSelected = domain.name === selected && hasBubbleChart(domain.name);
+    return measureSelected
+      ? [getLocalizedCategoryMetricName(measureSelected.metric)]
```

Comment:
Yeah, I was hesitating to choose between `getLocalizedMetricName` and `getLocalizedCategoryMetricName`. Well let's try `getLocalizedMetricName` since it's less misleading.